analytics/agma/README.md

```diff
@@ -0,0 +1,28 @@
+# agma Analytics
+
+In order to use the Agma Analytics Adapter, please adjust the accounts / endpoint with the data provided by agma (https://www.agma-mmc.de).
+
+## Configuration
+
+```yaml
+analytics:
+    agma:
+        # Required: enable the module
+        enabled: true
```

Comment:
The module should probably default to being disabled. You don't want a bunch of random PBS hosts trying to connect to your systems with the default parameters.

Reply:
Oh, yes, this is the README. I am thinking in terms of the bidder adapters that provide their own yaml files for configuration.